### PR TITLE
Add types definition for jstz (jsTimezoneDetect)

### PR DESCRIPTION
**Pull request by olamothe**
case 1. Add a new type definition.
- [X] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [X] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

Original jsTimezoneDetect project repository : https://bitbucket.org/pellepim/jstimezonedetect

### CODE REVIEW

File: jstz/jstz.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+interface JsTimezoneDetect {
+    determine: ()=> {
+      name: ()=> string
+    }
+}
+
+declare var jstz: JsTimezoneDetect;
+
+declare module "jstz" {
```
Comment (mhegazy) [inline jstz/jstz.d.ts]: the module name on npm is "jstimezonedetect". also the file name should be changed to `jstimezonedetect`

**Review by mhegazy — CHANGES_REQUESTED** [review]
Please change the file name and the module name.

File: jstz/jstz.d.ts (hunk 1 above)
Comment (olamothe) [inline jstz/jstz.d.ts]: Ahh, both are available on npm (jstz and jstimezonedetect), but you are right that jstz is an unofficial fork.

This is why I was confused.

I renamed everything to jstimezonedetect.

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**